go/kubebuilder/memcached-operator/README.md

```diff
@@ -41,16 +41,18 @@ Build the Memcached operator image and push it to a public registry, such as qua
 ```
 $ make install
 $ export IMG=quay.io/example-inc/memcached-operator:v0.0.1
-$ make docker-build $IMG
-$ docker push $IMG
-$ make deploy $IMG
+$ make docker-build IMG=$IMG
+$ docker push IMG=$IMG
 ```
 
 **NOTE** The `quay.io/example-inc/memcached-operator:v0.0.1` is an example. You should build and push the image for your repository.
+As this example showcases validation webhook creation, please follow [this][certmanager] guide to install cert-mamager into cluster prior to deployment.
```

Comment:
This note is specific to explain that `exemple-inc` is the registry and relevant to the build steps. 
The `certmanager ` is aa requirement step and should be done before. Could you please add a pre-requirement step/section with this info before the install/build/push?